src/main/java/net/vulkanmod/vulkan/VRenderSystem.java

```diff
@@ -154,12 +155,25 @@ public static MappedBuffer getShaderFogColor() {
         return shaderFogColor;
     }
 
-    public static void setClearColor(float f1, float f2, float f3, float f4) {
-        ColorUtil.setRGBA_Buffer(clearColor, f1, f2, f3, f4);
+    public static void setClearColor(float f0, float f1, float f2, float f3) {
```

Comment:
Why only change the params here just keep it the same

Reply:
The https://github.com/xCollateral/VulkanMod/pull/359 PR that this one is recycled from had some very badly formatted code, and I forgot to clean up the formatting issues as a result

Will force push the crash to correct them